app/serializers/financial_transaction_serializer.rb

```diff
@@ -0,0 +1,18 @@
+class FinancialTransactionSerializer < ActiveModel::Serializer
+  include ApplicationHelper
+
+  attributes :id, :user_id, :user_name, :amount, :note, :created_at
+
+  def user_name
+    show_user(object.user)
```

Comment:
what's our preferred stye? with or without brackets?

Reply:
Let's stick to without brackets in simple cases like this, with brackets in more complex cases (like nested calls and if statements etc.). Will change.